gef-ui/server/controllers/application-details/index.js

```diff
@@ -93,8 +93,8 @@ function buildBody(app, previewMode) {
     checkerCanSubmit: app.checkerCanSubmit,
     makerCanReSubmit: makerCanReSubmit(userSession, app),
     ukefDecision: app.ukefDecision,
-    isUkefReviewAvailable: isUkefReviewAvailable(app.status),
-    isUkefReviewPositive: isUkefReviewPositive(app.status),
+    isUkefReviewAvailable: isUkefReviewAvailable(app),
+    isUkefReviewPositive: isUkefReviewPositive(app),
```

Comment:
There should be either an updated test for this or a failing test 🤔

Reply:
Yep have updated the tests below, they were all failing should be passing now.